### PR TITLE
Cleanup, refactoring and issue fixes

### PR DESCRIPTION
**Pull request by wictorsson**
Cleanup, refactoring and issue fixes from the Trello board, post MVP.

### CODE REVIEW

File: src/pages/GroupSubPages/GroupMeetingsSubPage.js [hunk 1]
```diff
@@ -85,42 +85,35 @@ const GroupMeetingItem = ({ meeting, isNext }) => {
...
-            <div className="paragraph-md">Keine Termine</div>
-          </div>
+      <div className="flex items-center bg-BG_GRAY paragraph-lg ">
+        <div className="grid grid-cols-3 py-2 items-center px-2 border rounded-2xl md:border-none w-full">
+          <div className="paragraph-md">Keine Termine</div>
         </div>
       </div>
     );
   }
+  console.log(group);
```
Comment (meyerfr) [inline src/pages/GroupSubPages/GroupMeetingsSubPage.js:98]: remove console log

File: src/util/form.helper.js [hunk 2]
```diff
@@ -10,6 +10,12 @@ export const isEmail = (value) => {
 };
 export const required = (value) =>
   value ? undefined : "Das Feld darf nicht leer sein";
+
+export const minCharRequired = (value) =>
+  value && value.trim().length >= 3
+    ? undefined
+    : "Das Feld muss min. 3 Zeichen beinhalten";
+
```
Comment (meyerfr) [inline src/util/form.helper.js:18]: rewrite to be like function minLength or maxLength.

File: src/util/form.helper.js [hunk 3]
```diff
@@ -238,15 +245,15 @@ export const groupFields = {
     label: "Wie soll deine Gruppe heißen?",
     placeholder: "Name",
     hint: "Bitte gib den Gruppen Namen ein.",
-    validate: [required, (value) => maxLength(value, 70)],
+    validate: [minCharRequired, (value) => maxLength(value, 70)],
```
Comment (meyerfr) [inline src/util/form.helper.js:248]: keep required and simply add minCharRequired to array
```suggestion
    validate: [required, (value) => minCharRequired(value, 3), (value) => maxLength(value, 70)],
```

File: src/util/form.helper.js [hunk 4]
```diff
@@ -238,15 +245,15 @@ export const groupFields = {
...
   },
   description: {
     type: "textarea",
     name: "groupDescription",
     label: "Wie würdest du deine Gruppe beschreiben?",
     placeholder: "Kurze Gruppenbeschreibung",
     maxLength: 500,
-    validate: [required, (value) => maxLength(value, 500)],
+    validate: [minCharRequired, (value) => maxLength(value, 500)],
```
Comment (meyerfr) [inline src/util/form.helper.js:256]: same as above

File: src/mockDataSlice.js
Comment (meyerfr) [inline src/mockDataSlice.js:1]: Delete file

**Review by meyerfr — APPROVED** [review]

**Comment by wictorsson** [comment]
Merge after fixing code review comments

**Pull request merged**